_posts/2019-10-24-socratic-seminar-97.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+type: socratic
+title: "Socratic Seminar 97"
+meetup: https://www.meetup.com/BitDevsNYC/events/264604145/
+---
+
+Special thanks to Chaincode Labs and NYU Blockchain Digital Asset Forum for food, refreshments and event space.
+
+## Presentation
+
+- Max Galka - [Elementus](https://elementus.io/)
+
+## Mailing Lists and Bitcoin Optech
+
+#### Mailing Lists
+
+##### bitcoin-dev
+- [Taproot updates](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017378.html)
+- [Removal of reject network messages from Bitcoin Core (BIP61)](https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2019-October/017400.html)
+
+##### lightning-dev
+
+#### Optech
+
+- [Newsletter #64](https://bitcoinops.org/en/newsletters/2019/09/18/)
+- [Newsletter #65](https://bitcoinops.org/en/newsletters/2019/09/25/)
+- [Newsletter #66](https://bitcoinops.org/en/newsletters/2019/10/02/)
+- [Newsletter #67](https://bitcoinops.org/en/newsletters/2019/10/09/)
+- [Newsletter #68](https://bitcoinops.org/en/newsletters/2019/10/16/)
+
+## Network Data
+
+- [Six Addresses Are Dominating Ethereum (ETH) Usage Right Now](https://bitcoinist.com/six-addresses-are-dominating-ethereum-eth-usage-right-now/)
```

Comment:
I don't trust bitcoinist as a news source

Reply:
Alright let's remove it.